tests/core/test_collections.py

```diff
@@ -87,6 +88,7 @@ def test_collection_defaults_to_hook_config():
   """
     cm = CollectionManager()
     cm.create_collection("foo")
+    cm.get("foo").include_regex = "*"
```

Comment:
can the string be empty too? regex='*' would include ''

Reply:
The user cannot input an empty string, "*" however is accepted.